### PR TITLE
[BC-break] Twig 2.x compatibility

### PR DESCRIPTION
**Pull request by haroldiedema**
Breaking changes include but not limited to:

- PHP 7 (strict types)
- Dropped support for Twig 1.x
- Symfony 3.3+

### CODE REVIEW

File: src/Bundle/CacheWarmer/WebpackCompileCacheWarmer.php [hunk 1]
```diff
@@ -1,4 +1,5 @@
 <?php
+declare(strict_types = 1);
 namespace Hostnet\Bundle\WebpackBundle\CacheWarmer;
```
Comment (iltar) [inline src/Bundle/CacheWarmer/WebpackCompileCacheWarmer.php]: What about the copyright notices?

File: src/Component/Configuration/Loader/CssLoader.php [hunk 2]
```diff
@@ -1,12 +1,15 @@
...
+ */
 declare(strict_types = 1);
 namespace Hostnet\Component\Webpack\Configuration\Loader;
 
 use Hostnet\Component\Webpack\Configuration\CodeBlock;
 use Hostnet\Component\Webpack\Configuration\ConfigExtensionInterface;
 use Symfony\Component\Config\Definition\Builder\NodeBuilder;
 
-final class CSSLoader implements LoaderInterface, ConfigExtensionInterface
+final class CssLoader implements LoaderInterface, ConfigExtensionInterface
```
Comment (iltar) [inline src/Component/Configuration/Loader/CssLoader.php]: Don't forget to mention this BC break when releasing the next major

Comment (haroldiedema) [inline src/Component/Configuration/Loader/CssLoader.php]: :+1:

File: composer.json [hunk 3]
```diff
@@ -1,34 +1,35 @@
...
+{
+    "name":        "hostnet/webpack-bundle",
+    "type":        "symfony-bundle",
+    "description": "Integrates Webpack with Symfony2",
+    "license":     "MIT",
+    "require": {
+        "php":                    "^7.0",
+        "symfony/monolog-bundle": "^3.1.0",
+        "symfony/symfony":        "3.3.*,>=3.3.2",
+        "twig/twig":              "^2.4.3"
```
Comment (yannickl88) [inline composer.json]: Do you really need twig 2.4.3? If possible, I would try to put it as low as possible (perhaps 2.0.0) to ensure maximum compatibility. Not everyone is in fully up to date.

Same with Symfony, depending on 3.0.0 or higher seems to me a better options unless you really need those features/patches in the later versions.

File: composer.json [hunk 4]
```diff
@@ -1,34 +1,35 @@
...
+    },
+    "require-dev": {
+        "hostnet/phpcs-tool":     "^4.1.4",
```
Comment (yannickl88) [inline composer.json]: is there a reason not to use 5.2.0?

File: .travis.yml [hunk 5]
```diff
@@ -14,14 +14,7 @@ cache:
...
   include:
-    # Symfony 2.7.0
-#    - php: 5.5
-#      env: COMPOSER_FLAGS="--prefer-lowest"
-    # Symfony 2.8
-#    - php: 5.6
-#      env: SYMFONY_VERSION="2.8.*@dev"
-    # Symfony 3.0
-    - php: 5.6
+    - php: 7.0
```
Comment (yannickl88) [inline .travis.yml]: can you add a `env: COMPOSER_FLAGS="--prefer-lowest"` option too?

File: composer.json (hunk 4 above)
Comment (haroldiedema) [inline composer.json]: Rule exclusion doesn't seem to work properly / `phpcs.xml.dist` isn't read from the repository root.

File: composer.json (hunk 3 above)
Comment (iltar) [inline composer.json]: Has to be verified, not sure which version introduced the namespaces but it was somewhere 20 days ago

Comment (iltar) [inline composer.json]: Also, any specific reason why to run higher than 3.3.2? What's wrong with 3.3.0 or 3.0.0?

File: composer.json (hunk 4 above)
Comment (yannickl88) [inline composer.json]: I do not have this issue when I checkout your branch. Even in the current state with 5.2.0 there are not PHPCS errors.

Comment (yannickl88) [inline composer.json]: http://imgur.com/a/wq6va

File: .travis.yml (hunk 5 above)
Comment (haroldiedema) [inline .travis.yml]: done.

File: .travis.yml [hunk 6]
```diff
@@ -14,16 +14,12 @@ cache:
...
       env: DEPS=dev
 
+env:
+  - COMPOSER_FLAGS="--prefer-lowest"
```
Comment (yannickl88) [inline .travis.yml]: please do this in the matrix. see https://github.com/yannickl88/form-handler-bundle/blob/master/.travis.yml

you are starting only one build now. what I would like:
```yaml
env:
  - COMPOSER_FLAGS="--prefer-stable"

php:
    - '7.0'
    - '7.1'
    - nightly

matrix:
  include:
    - php: 7.0
      env: COMPOSER_FLAGS="--prefer-lowest"
    - php: 7.0
      env: COMPOSER_FLAGS=""
    - php: 7.1
      env: COMPOSER_FLAGS="--prefer-lowest"
    - php: 7.1
      env: COMPOSER_FLAGS=""
  allow_failures:
    - php: nightly
```

**Pull request merged**